xwiki-platform-core/xwiki-platform-web/src/main/webapp/resources/js/xwiki/actionbuttons/actionButtons.js

```diff
@@ -384,11 +384,12 @@ require(['jquery'], function ($) {
 
   $(window).on("scroll resize load click xwiki:dom:refresh", function() {
     var isFullScreen = $('.fullScreenWrapper').length > 0
+    var isVisible = $container.css('display') != 'none';
```

Comment:
Why not https://api.jquery.com/visible-selector/ ?

    var isVisible = $container.is(':visible');

Reply:
It's even better, thanks.